.github/workflows/merge-check.yml

```diff
@@ -0,0 +1,32 @@
+name: Merge checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  gradle:
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, 'ci skip') }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Cache Gradle Caches
+        uses: actions/cache@v1
```

Comment:
Looks like cache@v2 is the latest. Any reason not to use that?

Reply:
Nope. Copied from an example using cache@v1